Shared/Extensions/URLExtensions.swift

```diff
@@ -556,6 +556,12 @@ private extension URL {
             return ""
         }
 
+        var currentHost = host
+        // Hosts can have a trailing period after the tld
+        if host.hasSuffix(".") {
+            currentHost = String(currentHost.dropLast())
```

Comment:
Swift-tip: Since `currentHost` is mutable, you can just do `currentHost.removeLast()` instead of assigning it a copy 🙂